shop/src/app/app.component.ts

```diff
@@ -1,10 +1,13 @@
 import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'shop';
+  title: string = 'shop';
+
+  constructor (private http: HttpClient) {}
```

Comment:
Is the httpClient service really needed here? if it's not used, it can be removed

Reply:
Done